src/NuGet.Core/NuGet.Protocol/Converters/SafeBoolConverter.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
BOMs.

Use `git checkout dev -- src/NuGet.Core/NuGet.Protocol/Converters/SafeBoolConverter.cs ` from the root.

Reply:
Reverted.